src/libs/API.js

```diff
@@ -547,27 +547,27 @@ function Report_TogglePinned(parameters) {
 
 /**
  * @param {Object} parameters
+ * @param {Number} parameters.accountID
```

Comment:
Also @sketchydroide Can you switch these methods back to the order they were in? It's making the diff bigger unnecessarily.